pyteal/ast/jsonref.py

```diff
@@ -0,0 +1,77 @@
+from typing import TYPE_CHECKING
+from enum import Enum
+
+from pyteal.types import TealType, require_type
+from pyteal.errors import TealTypeError, verifyFieldVersion, verifyTealVersion
+from pyteal.ir import TealOp, Op, TealBlock
+from pyteal.ast.expr import Expr
+from pyteal.ast.leafexpr import LeafExpr
+
+if TYPE_CHECKING:
+    from pyteal.compiler import CompileOptions
+
+
+class JsonRefType(Enum):
+    # fmt: off
+    #              id |    name   |      type      | min version
+    json_string = (0, "JSONString", TealType.bytes,  7)
+    json_uint64 = (1, "JSONUint64", TealType.uint64, 7)
+    json_object = (2, "JSONObject", TealType.bytes,  7)
```

Comment:
Since these are already namespaced under JsonRefType does it make sense to omit `json_`?

Reply:
if you implement the methods for these instead, the naming matters less I think